script.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+
+const mainnetPath = path.join(__dirname, 'chains/mainnet');
+const files = fs.readdirSync(mainnetPath);
+
+files.forEach(file => {
+
+    const filePath = path.join(mainnetPath, file);
+    if (filePath.endsWith('disabled')) return
+    const data = require(filePath);
+    axios.get(data.lcd + '/cosmos/gov/v1/proposals?pagination.limit=9')
```

Comment:
rather than just checking the proposals endpoint, please use the `node_info` endpoint to derive actual version.

https://phoenix-lcd.terra.dev/swagger/#/Service/GetNodeInfo